src/Cart/CartScreen.js

```diff
@@ -23,18 +50,109 @@ const styles = StyleSheet.create({
     marginTop: 10,
     backgroundColor: '#0492c2',
   },
+  scrollView: {
+    flex: 1,
+    width: '95%',
+    marginBottom: '2%',
+  },
+  subtotalContainer: {
+    flexDirection: 'row',
+    marginBottom: '4%',
+    marginTop: '0%',
+    marginLeft: '2%',
+    marginRight: '2%',
+  },
+  subtotalText: {
+    fontWeight: 'normal',
+    fontSize: 18,
+    marginLeft: '4%',
+  },
+  subtotal: {
+    marginRight: '4%',
+    marginLeft: '4%',
+    fontWeight: 'normal',
+    fontSize: 18,
+  },
 });
 
+// const fakeItems = { name: 'daikon', quantity: '2' };
+
 export default function CartScreen({ navigation }) {
+  const [itemList, setItemList] = useState([]);
+  const [refresh, setRefresh] = useState(0);
+  const [subtotal, setSubtotal] = useState(0);
+
+  const getItems = () => {
+    const list = [];
+    base('Cart').select({ filterByFormula: "({users}='helen@gmail.com')" }).eachPage((records, fetchNextPage) => {
```

Comment:
Can you mark this w/ a TODO comment to change later when sign in is implemented?